src/plugins/kibana_react/public/table_list_view/actions.ts

```diff
@@ -47,7 +46,7 @@ export interface OnClickDeleteItemsAction {
 }
 
 export interface OnFilterChangeAction {
-  type: 'onFilterChange';
+  type: 'onSearchQueryChange';
```

Comment:
Why this change?

Reply:
It better reflects what is happening IMO. Filter change will happen in different ways with the new design (e.g. clicking a tag inside the table) so I thought it was a better naming.